src/ng-context-menu.js

```diff
@@ -40,12 +40,22 @@
                   (doc.clientTop || 0),
                 elementWidth = menuElement[0].scrollWidth,
                 elementHeight = menuElement[0].scrollHeight;
+              var pageX;
+              var pageY;
+              if (event.pageX || event.pageY) {
```

Comment:
Can you put in comment for future readers just to explain why this is here please?